pages/index.js

```diff
@@ -3,27 +3,30 @@ import InterviewCard from "@/components/InterviewCard";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import { interviews } from "@/public/interviews";
+import Footer from "@/components/Footer";
 
 export default function HomePage() {
   const router = useRouter();
   // router.push("/interview");
```

Comment:
You're currently not using the `router`. Best remove these lines and the import as well. Add it back in when you're ready to fully integrate it.